tests/test_admin.py

```diff
@@ -0,0 +1,231 @@
+import os
+from datetime import datetime
+from distutils.util import strtobool
+
+from selenium import webdriver
+from wizard_builder.models import (
+    Checkbox, Choice, Conditional, Date, FormQuestion, MultiLineText,
+    MultipleChoice, PageBase, QuestionPage, RadioButton, SingleLineText,
+    SingleLineTextWithMap, TextPage,
+)
+
+from django.contrib.auth import get_user_model
+from django.contrib.sites.models import Site
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.test import override_settings
+
+try:
+    from urlparse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
+
+
+User = get_user_model()
+
+SCREEN_DUMP_LOCATION = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), 'screendumps'
+)
+
+
+class FunctionalTest(StaticLiveServerTestCase):
```

Comment:
I mean, ok, in working on that I got the entire tox build matrix passing on local, which was kind of cool I guess